README.md

```diff
@@ -295,6 +295,97 @@ To list open replica version management proposals:
 ./scripts/list-open-rvm-proposals.sh
 ```
 
+### Creating closed proposals
+
+#### Creating proposals in the past
+
+1. Check your current system time:
+   ```bash
+   date
+   ```
+2. Set your system time to be at least 48 hours in the past:
+   ```bash
+   sudo date -s "Feb 24 20:55:54 CET 2024"
```

Comment:
Is this something one wants to do on their computer? I personally wouldn't like to change the date of my computer manually.

Is there a way to do it only for that specific terminal session or so?

Reply:
I have an idea to make this work with a Docker container. I'll handle this in a separate PR though if you don't mind.